include/taskolib/deserialize_sequence.h

```diff
@@ -84,26 +84,24 @@ std::istream& operator>>(std::istream& stream, Step& step);
  * the time on loading the step.
  * \note the collection of context variable names can also be an empty list, ie. \c [] .
  *
- * \param path  the path to a file from which the step should be loaded
+ * \param path from which the Step should be loaded.
```

Comment:
Personally I still think of a directory when I just read "path", so I liked the old version with "file" better. But maybe that's just me...

Reply:
Well, just `filename`? That can usually contain an absolute or relative path.

Reply:
`const std::filesystem::path& path`

`path` as variable name here is just a technical type description.
The variable name should describe what is in it.
So maybe `stepfile` or `filename` or whatever?

Reply:
> Personally I still think of a directory when I just read "path", so I liked the old version with "file" better. But maybe that's just me...

Hmmm, after a second contemplation I agree to your thoughts. I will try to rename each "path" to "folder" and use an appropriate wording.